### PR TITLE
The block attacher helm deployment process is changed helm chart path is retrieved dynamically  

### PR DESCRIPTION
**Pull request by mumutyal**
e2e changes to align to travis build format

### CODE REVIEW

File: block-storage-attacher/tests/e2e/scripts/common.sh [hunk 1]
```diff
@@ -507,6 +507,11 @@ function install_blockvolume_plugin {
         echo "helm chart not found. Hence exiting"
         exit 1
     fi
+	PLUGIN_BUILD=$(git ls-remote --tags  | grep -o 'v.*' | sort -r | head -1)
+        RELEASE_TAG=$(git ls-remote --tags  git@github.ibm.com:alchemy-containers/block-storage-attacher.git| grep -o 'v.*' | sort -r | head -1)
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/scripts/common.sh:511]: You need to pick the helm chart from here https://github.ibm.com/alchemy-containers/block-storage-attacher/tree/v2.0.2_83 which is internal repo. Please check in this link first to find the helm chart then make changes to PR.

**Review by akgunjal — CHANGES_REQUESTED** [review]
Address comments

File: block-storage-attacher/tests/e2e/scripts/common.sh (hunk 1 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/scripts/common.sh:511]: Yes We are cloning the helm chart form the same path  only

git clone -b ${RELEASE_TAG}  "git@github.ibm.com:alchemy-containers/block-storage-attacher.git" $CLONE_PATH

**Review by akgunjal — APPROVED** [review]
LGTM

**Pull request merged**